worklist/views.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from django.shortcuts import render
+from worklist.models import WorkList
+
+
+# View to create a worklist
+def create_worklist(request):
+    content = {}
+    if request.method == 'POST':
+        content = {'error': 0, 'message': 'Successfully saved worklist'}
+        data = {'name': request.POST.get('name', ''),
+                'theme': request.POST.get('theme', ''),
+                'description': request.POST.get('description', ''),
+                'created_by': request.POST.get('created_by', ''),
+                'psid': request.POST.get('psid', ''),
+                'psid_added_by': request.POST.get('created_by', '')}
+
+        if data['name'] and WorkList.objects.filter(name=data['name']).exists():
+            content['error'] = 1
+            content['message'] = 'Worklist with this name already exists'
+            render(request, 'worklist-page.html', content)
+        WorkList.create_object(data)
+    return render(request, 'create-worklist.html', {'content':content})
+
+
+def show_worklist(request):
+    searchTerm = request.GET.get('searchTerm','')
```

Comment:
`searchTerm` made me think we're searching through names (with substring match). Renaming this to `name` would make it more obvious that we're just retrieving a worklist by name.

Reply:
changed that :).